test/BaselineOutput/SingleDebug/SDCALR/BinarySDCA-CV-breast-cancer-out.txt

```diff
@@ -1,4 +1,4 @@
-maml.exe CV tr=SDCA{maxIterations=5 checkFreq=9 nt=1} threads=- dout=%Output% data=%Data% seed=1
+maml.exe CV tr=SDCALR{maxIterations=5 checkFreq=9 nt=1} threads=- dout=%Output% data=%Data% seed=1
```

Comment:
>SDCALR [](start = 15, length = 6)

So you branch trainer from SDCA to SDCALR and SDCA. 
Why you get same results for SDCALR as for SDCA before?
Would you get different results if you leave SDCA here?

Reply:
Previous SDCA with default settings is exactly the new SDCALR.

---
In reply to: [256212640](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256212640) [](ancestors = 256212640)

Reply:
So maybe you leave SDCA as it is, and rename your current SDCA to something else?

---
In reply to: [256219306](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256219306) [](ancestors = 256219306,256212640)

Reply:
I somehow wants to have SDCALR instead of SDCA, if it doesn't cause too many troubles. To my understanding, when mentioning SDCA, the default loss used should be hinge loss instead of logistic one.

---
In reply to: [256219554](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256219554) [](ancestors = 256219554,256219306,256212640)

Reply:
Only reason why I bother you is if left thing like they are right now, one day we will get mail what my script used to work fine, but now it produce something different and unexpected.
and reason would be old command line where people specify SDCA, and get different learner than before.

---
In reply to: [256226838](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256226838) [](ancestors = 256226838,256219554,256219306,256212640)

Reply:
In old world, specifying SDCA doesn't always mean the new SDCALR. It can be just new SDCA as well. Users may send out emails if they use SVM.

---
In reply to: [256229935](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256229935) [](ancestors = 256229935,256226838,256219554,256219306,256212640)

Reply:
We will have BC in terms of command line tools in the next iteration.

---
In reply to: [256261914](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256261914) [](ancestors = 256261914,256229935,256226838,256219554,256219306,256212640)